BookingsApi/BookingsApi/Controllers/V1/WorkHoursController.cs

```diff
@@ -198,33 +198,51 @@ public async Task<IActionResult> UpdateVhoNonAvailabilityHours(string username,
             
             return NoContent();
         }
-        
+
         /// <summary>
         /// Delete non availability work hours for vho
         /// </summary>
-        /// <param name="id"></param>
+        /// <param name="username">the justice user username</param>
+        /// <param name="nonAvailabilityId">Identifier for the non-available period</param>
         /// <returns>vho with list of non availability work hours</returns>
-        [HttpDelete("/NonAvailability")]
+        [HttpDelete("/NonAvailability/VHO/{username}/{nonAvailabilityId}")]
```

Comment:
just a few points:

- is it necessary to pass in justice username and non availability id? You can get the justice user from the non availability
- wouldn't justice user id be a better identifier than username?
- shouldn't this go into V2 as it's a breaking change?

Reply:
let's get on a call about this one